specs/commandline/verify.md

```diff
@@ -197,3 +197,47 @@ export NOTATION_EXPERIMENTAL=1
 # The value of --scope should be set base on the trust policy configuration
 notation verify --oci-layout --scope "local/hello-world" hello-world:v1
 ```
+
+### Verify signatures on an OCI artifact with json output
+
+Use the `--output` flag to format verification output in json.
+
+```shell
+notation verify --output json localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+```
+
+An example of output messages for a successful verification:
+
+```text
+{
+    "reference": "localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9",
+    "userMetadata": {
+        "io.wabbit-networks.buildId": "123"
+    },
```

Comment:
Should we also add signature reference/digest? And move userMetadata to be a child of it?
```
{
    "reference": "localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9",
    "signatures":  [
        {
            "digest": "sha256:73c803930ea3ba1e54bc25c2bdc53edd0284c62ed651fe7b00369da519a33",
            "userDefinedAttributes":  {
                "io.wabbit-networks.buildId": "123" 
            }
....
```

Also to be consistent with [inspect](https://github.com/notaryproject/notation/blob/main/specs/commandline/inspect.md#inspect-signatures-on-the-supplied-oci-artifact-with-an-example-of-json-output), should we rename `userMetadata` to `userDefinedAttributes`